erpnext/maintenance/doctype/maintenance_schedule/maintenance_schedule.py

```diff
@@ -150,7 +150,7 @@ def validate_maintenance_detail(self):
 			elif not d.no_of_visits:
 				throw(_("Please mention no of visits required"))
 			elif not d.sales_person:
-				throw(_("Please select Incharge Person's name"))
+				throw(_("Please select Incharge Person's name for item: {0}".format(d.item_name)))
```

Comment:
I think a better wording would be:

```suggestion
				throw(_("Please select an Incharge Person for item: {0}".format(d.item_name)))
```